molo/core/content_import/views.py

```diff
@@ -7,13 +7,28 @@
 from rest_framework.permissions import IsAuthenticated
 from rest_framework.decorators import parser_classes
 
-from molo.core.content_import import api
 from unicore.content.models import Localisation
 
+from molo.core.content_import import api
+from molo.core.content_import.errors import SiteResponseError
+
 
 @api_view(['GET'])
-def get_repos(request):
-    return Response({'repos': api.get_repo_summaries()})
+def get_repo_summaries(request):
+    params = request.query_params
```

Comment:
I thought the UI was just going to be a text box field. Is the expectation that the JS will parse the URL before making the api call? Why not just accept a URL here?
Sorry, I think I'm getting confused by why you need to parse the URL and not just pass the user submitted URL directly.